packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -132,6 +132,15 @@ export type PaymentBillingDetails = ReturnType<
   billing_agreement_id?: string;
 };
 
+// The countries we need region data for
+export const COUNTRIES_LONG_NAME_TO_SHORT_NAME_MAP = {
+  // The long name is used in the BigQuery metrics logs; the short name is used
+  // in the Stripe customer billing address.  The long names are also used to
+  // index into the country to states maps.
+  'United States': 'US',
+  Canada: 'CA',
+} as { [key: string]: string };
+
```

Comment:
Is this used at all in `stripe.ts`?

Reply:
It's used in the route handlers (`routes/subscriptions/{paypal,stripe}.ts`